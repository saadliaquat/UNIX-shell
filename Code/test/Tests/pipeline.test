ls | grep .json
ls -al | grep total | cut -d ' ' -f 2
cat config.json | grep -oE "easy|medium|advanced" | sort | uniq -c | sort -n
find Tests/ -maxdepth 1 -type f | xargs wc -l | sort -n
ps -opid,comm | grep -v PID | grep bash | awk '{print $1}' | sort | head -1