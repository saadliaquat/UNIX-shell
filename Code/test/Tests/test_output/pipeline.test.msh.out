config.json
config.json.log
16
      3 advanced
      3 easy
      3 medium
   2 Tests/wild_chaining.test
   2 Tests/wildcards.test
   4 Tests/pipeline.test
   5 Tests/chaining.test
   6 Tests/quotes.test
   7 Tests/builtins.test
  12 Tests/ioredir.test
  13 Tests/simple.test
  19 Tests/aliases.test
  19 Tests/aliases.test.custom
  89 total
13
